.gitignore

```diff
@@ -0,0 +1,41 @@
+*#
+*.iml
+*.ipr
+*.iws
+*.jar
+*.sw?
+*~
+.#*
+.*.md.html
+.DS_Store
+.classpath
+.factorypath
+.gradle
+.idea
+.metadata
+.project
+.recommenders
+.settings
+.springBeans
+/code
+MANIFEST.MF
+_site/
+activemq-data
+bin
+build
+!/**/src/**/bin
+!/**/src/**/build
+build.log
+dependency-reduced-pom.xml
+dump.rdb
+interpolated*.xml
```

Comment:
Do we need all of this ignors?

Reply:
This is the standard set of ignores provided by the Spring Boot team: https://github.com/spring-projects/spring-boot/blob/master/.gitignore.
I suggest that these files can be generated by the spring boot framework.

Reply:
But this sample doesn't have all features  that https://github.com/spring-projects/spring-boot/blob/master has, so please leave only necessary one